infrastructure/bicep/core/modules/virtualNetworks.bicep

```diff
@@ -0,0 +1,69 @@
+// --- PARAMETERS --- //
+@description('Azure Location/Region')
+param location string = resourceGroup().location
+
+@description('Name of the virtual network')
+param virtualNetworkName string
+
+@description('Name of the SQL DB subnet')
+param sqlDBSubNetName string
+
+@description('Name of the VM subnet')
+param vmSubNetName string
+
+@description('Id of the network security group')
+param networkSecurityGroupId string
+
+// --- RESOURCES --- //
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2024-10-01' = {
+  name: virtualNetworkName
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        '10.0.0.0/16'
+      ]
+    }

```

Comment:
We should use a different address space than the existing VNet (so they can be peered)